app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,71 @@
+class TweetsController < ApplicationController
+  #index
+  get '/tweets' do
+    if Helper.logged_in?(session)
+      @tweets= Tweet.all
+      @user = Helper.current_user(session)
+      erb :'tweets/tweets'
+    else
+      redirect to "/login"
+     end
+  end
+
+  #new
+  get '/tweets/new' do
+    if Helper.logged_in?(session)
+      erb :'tweets/new'
+    else
+      redirect "/login"
+    end
+  end
+
+  #create
+  post '/tweets' do
+    user = Helper.current_user(session)
+    if !params[:content].empty?
+      @tweet= Tweet.create(content: params[:content])
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: user)
```